Resources/Textures/Objects/Specific/Chemistry/chem_bag.rsi/meta.json

```diff
@@ -0,0 +1,18 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Taken from paradise station https://github.com/ParadiseSS13/Paradise and rescaled by reapsnI",
```

Comment:
link the commit you took it (and the dmi itself in said commit if possible)

Reply:
@reapsnI Could you look that up? Or do you roughly know what date you downloaded it from paradise?

Reply:
Anyway, looked up the last commit in Paradise before @reapsnI made his commit as a reasonable estimate. Would a good string be `"Taken from ParadiseSS13/Paradise at https://github.com/ParadiseSS13/Paradise/blob/ebf9a7b5e89f5523c5e1c3d2692bf54dac322cae/icons/obj/chemical.dmi and rescaled by reapsnI"`? @AJCM-git